Formula/wiimms-szs-tools.rb

```diff
@@ -0,0 +1,29 @@
+class WiimmsSzsTools < Formula
+  desc "Wiimms SZS Toolset is a set of command-line tools to manipulate Wii files"
+  homepage "https://szs.wiimm.de/"
+  url "https://download.wiimm.de/source/wiimms-szs-tools/wiimms-szs-tools.source-2.19a.txz"
+  license "GPL-2.0-or-later"
+
+  head do
+    url "https://github.com/Wiimm/wiimms-szs-tools.git"
+  end
+  depends_on "libpng" => :build
+  uses_from_macos "ncurses"
+
+  def install
+    cd "project" do
+      system_command "/usr/bin/sed",
```

Comment:
This would be better as a patch upstream to allow `DESTDIR` to override this. It's how most other projects work with make.

Reply:
I'm not sure how to do that.

Reply:
I think @SMillerDev is saying to raise this issue with the upstream project, so that it gets fixed for everyone.

Reply:
I don't think the guy who owns the GitHub repository uses its features to be honest, it's kind of introverted. He's also a busy person. I don't think he would see a PR, likewise merge it.

Reply:
That would be a bigger blocker for me than the notability. We need to be able to rely on software being maintained and supported to ship it in homebrew. https://docs.brew.sh/Acceptable-Formulae#niche-or-self-submitted-stuff

Reply:
Well the software is maintained, but I think the guy mostly keeps to himself on this project.

Reply:
If they don't accept contributions I wouldn't really consider it open source. Either way, it should be submitted upstream or Homebrew can't merge it.

Reply:
Sorry to ask but how do I make it upstream? Do I make a fork of it and then modify that and then modify the formula to use my fork?

Reply:
You make a fork and then you make a pull request to the original repository to include your changes. Then you point the formula to the new release that includes your changes.

Reply:
OK. The maintainer added some more commits to the repository that have to do with Mac builds and now compiling results in an error. For now I'll open an issue.

Reply:
https://github.com/Wiimm/wiimms-szs-tools/issues/1

Reply:
https://github.com/Wiimm/wiimms-szs-tools/issues/1